### PR TITLE
added a HOST variable to docker containers

### PR DESCRIPTION
**Pull request by mjksmith**
Docker containers created by local_run will now have a $HOST variable containing the hostname of the machine the container is running on.

### CODE REVIEW

File: paasta_tools/cli/cmds/local_run.py [hunk 1]
```diff
@@ -566,6 +569,11 @@ def get_instance_config(service, instance, cluster, soa_dir, load_deployments=Fa
     )
 
 
+def get_hostname():
+    (_, hostname) = _run('hostname')
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/local_run.py]: Heh, socket.getfqdn would be ok

File: paasta_tools/cli/cmds/local_run.py [hunk 2]
```diff
@@ -352,13 +352,14 @@ def get_container_name():
...
 
-def get_docker_run_cmd(memory, random_port, container_name, volumes, env, interactive, docker_hash, command):
+def get_docker_run_cmd(memory, random_port, container_name, volumes, env, interactive, docker_hash, command, hostname):
     cmd = ['docker', 'run']
     for k, v in env.iteritems():
         cmd.append('--env=\"%s=%s\"' % (k, v))
     # We inject an invalid port as the PORT variable, as marathon injects the externally
     # assigned port like this. That allows this test run to catch services that might
     # be using this variable in surprising ways. See PAASTA-267 for more context.
+    cmd.append('--env=HOST=%s' % hostname)
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/local_run.py]: This is all overkill. Just put socket.gethostname() right here and you are done.

File: paasta_tools/cli/cmds/local_run.py [hunk 3]
```diff
@@ -352,13 +352,14 @@ def get_container_name():
     return 'paasta_local_run_%s_%s' % (get_username(), randint(1, 999999))
 
 
-def get_docker_run_cmd(memory, random_port, container_name, volumes, env, interactive, docker_hash, command):
+def get_docker_run_cmd(memory, random_port, container_name, volumes, env, interactive, docker_hash, command, hostname):
     cmd = ['docker', 'run']
     for k, v in env.iteritems():
         cmd.append('--env=\"%s=%s\"' % (k, v))
     # We inject an invalid port as the PORT variable, as marathon injects the externally
```
Comment (solarkennedy) [inline paasta_tools/cli/cmds/local_run.py]: This comment is now kinda on the wrong line, try to keep this comment and the --env=port together

**Comment by Rob-Johnson** [comment]
lg2m. :ship:

**Pull request merged**